akka-stream/src/main/scala/akka/stream/extra/Timed.scala

```diff
@@ -24,8 +28,8 @@ private[akka] trait TimedOps {
   def timed[I, O, Mat, Mat2](source: Source[I, Mat], measuredOps: Source[I, Mat] ⇒ Source[O, Mat2], onComplete: FiniteDuration ⇒ Unit): Source[O, Mat2] = {
```

Comment:
Let's deprecate it right away `@deprecated("Moved to akka-stream-contrib project", since = "2.4.5")`